testdata/google/events/cloud/storage/v1/StorageObjectData-complex.json

```diff
@@ -0,0 +1,31 @@
+{
+  "bucket": "sample-bucket",
+  "cacheControl": "in nostrud dolore enim adipisicing",
+  "componentCount": 2239872,
+  "contentDisposition": "cupidatat aliqua exercitation",
+  "contentEncoding": "gzip",
+  "contentLanguage": "ut incididunt",
+  "contentType": "esse commodo",
+  "crc32c": "AAAAAA==",
+  "customerEncryption": { "keySha256": "est elit", "encryptionAlgorithm": "enim mollit" },
+  "etag": "COu8mb3Dn+kCEAE=",
+  "eventBasedHold": true,
+  "generation": 96883251,
+  "id": "sample-bucket/MyFile/1588778055917163",
+  "kind": "storage#object",
+  "kmsKeyName": "nulla",
+  "md5Hash": "id eiusmod",
```

Comment:
Possibly use a valid MD5 hash, base64-encoded? Example: "xrX0h3SqCoeoKidv+GvlBw=="
(That makes it obvious that the hash is expected to be in base64, not in hex.)

Reply:
sgtm